server/ngwmn/confluence_utils.py

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
I trust all the secret incantations in this module 😄 

But I think it would make sense for this to go in the ngwmn.services package.

Reply:
Yes it is a little mysterious! It is basically a copy of the same method in WQP although I did add the tests. The reason I have it separate is that it's not ngwmn specific. Not really a good reason I suppose.